groupcachepb/groupcache.proto

```diff
@@ -34,6 +35,7 @@ message SetRequest {
   required string key = 2;
   optional bytes value = 3;
   optional int64 expire = 4;
+  optional bool hotCache = 5;
```

Comment:
I personally would not make this variable optional considering it's a bool.

Reply:
Done so with proto3 update